### PR TITLE
Bumped golangci-lint from v1.56.2 to v1.61.0

### PR DESCRIPTION
**Pull request by PMaynard**
## Description

Bumped golangci-lint from v1.56.2 to v1.61.0

## Dependencies

n/a

## Checklist

Sorry, just fixing local build errors.

### CODE REVIEW

File: .bingo/Variables.mk [hunk 1]
```diff
@@ -29,11 +29,11 @@ $(BINGO): $(BINGO_DIR)/bingo.mod
 	@echo "(re)installing $(GOBIN)/bingo-v0.5.1"
 	@cd $(BINGO_DIR) && $(GO) build -mod=mod -modfile=bingo.mod -o=$(GOBIN)/bingo-v0.5.1 "github.com/bwplotka/bingo"
 
-GOLANGCI_LINT := $(GOBIN)/golangci-lint-v1.56.2
+GOLANGCI_LINT := $(GOBIN)/golangci-lint-v1.61.0
```
Comment (jotak) [inline .bingo/Variables.mk:32]: this is an auto-generated file (see header), it's not how it should be updated.
TBH I'm not a fan of "bingo", the tool used for bin dependencies here (I find it a bit opaque for the simple job we're asking .. and I always forget how it works :-/ ) .. let me see if I can quickly remove that and align this repo we our other ones

**Review by jotak — CHANGES_REQUESTED** [review]
Thanks @PMaynard 
.. but this probably won't do what you expect, this is a generated file that you changed

**Comment by jotak** [comment]
now there's a bunch of lint errors to fix 😊

**Comment by jotak** [comment]
I'm working on fixing the issues, will update soon

**Review by jotak — APPROVED** [review]

File: .bingo/Variables.mk (hunk 1 above)
Comment (PMaynard) [reply .bingo/Variables.mk]: ops, my bad. Good choice removing bingo.

**Pull request merged**